### PR TITLE
feat(ui): Add file icons to release detail changed files

### PR DESCRIPTION
**Pull request by matejminar**
Before:
image

After:
image

### CODE REVIEW

**Review by priscilawebdev — APPROVED** [review]
I love it! 👏

File: src/sentry/static/sentry/app/components/fileIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {IconFile} from 'app/icons';
+import theme from 'app/utils/theme';
+
+const FILE_EXTENSION_TO_ICON = {
+  jsx: 'react',
+  tsx: 'react',
+  js: 'javascript',
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/fileIcon.tsx:9]: Should `ts` go to javascript too?

**Comment by benvinegar** [comment]
This is awesome? Uh, can I also convince you to add a platform icon to the global project selector? *ducks*

File: src/sentry/static/sentry/app/components/fileIcon.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/fileIcon.tsx:9]: Yeah, I mapped it. I would love a dedicated TS icon though 🙂

**Comment by matejminar** [comment]
@benvinegar sure thing, I will submit new PR for that 👍

**Review by markstory — APPROVED** [review]

**Pull request merged**